services/app-web/src/components/Form/ErrorSummary/ErrorSummary.tsx

```diff
@@ -0,0 +1,91 @@
+import classNames from 'classnames'
+import classnames from 'classnames'
+import React, { useEffect, useRef } from 'react'
+import styles from './ErrorSummary.module.scss'
+
+type ErrorSummaryMessageProps = {
+    errorKey: string,
+    message: string
+}
+const ErrorSummaryMessage = ({
+    errorKey, message
+}: ErrorSummaryMessageProps): React.ReactElement => {
+    return (
+        <button
+            type="button"
+            className={classnames("usa-error-message", "usa-alert__text", styles.message)}
+            onClick={() => {
+                const fieldElement: HTMLElement | null = document.querySelector(
+                    `[name="${errorKey}"]`
+                );
+            
+                if (fieldElement) {
+                    fieldElement.focus();
+                }
+            }}
+        >
+        {message}
+      </button> 
+    ) 
+}
+
+function summaryHeading(numberOfErrors: number) {
+    if (numberOfErrors === 1) {
+        return "There is 1 error on this page";
+    } else {
+        return `There are ${numberOfErrors} errors on this page`;
+    }
+}
+
+function flattenMessage(message: string | string[]) {
+    if (typeof message === "string") {
+        return message;
+    }
+
+    return message.join(" and ");
+}
+
+export type ErrorSummaryProps = {
```

Comment:
This was heavily based on a similar component from EASi: https://github.com/CMSgov/easi-app/blob/master/src/components/shared/ErrorAlert/index.tsx